### PR TITLE
Updating getting-started

### LINKED ISSUE #62: Rename repo and update docs
As a developer, I'd like to rename the repo from `spring-cloud-dataflow-admin-kubernetes` to `spring-cloud-dataflow-server-kubernetes`.

### PR DESCRIPTION
**Pull request by trisberg**
Resolves #62

### CODE REVIEW

**Comment by cppwfs** [comment]
index.adoc title needs to change from "Spring Cloud Data Flow Admin for Kubernetes" to "Spring Cloud Data Flow Server for Kubernetes"

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -6,25 +6,23 @@ In this getting started guide, the Data Flow Server is run as a standalone appli
...
 +
-The redis service will be used for messaging between modules in the stream.  There are sample replication controller and service YAML files in the `spring-cloud-dataflow-admin-kubernetes` repository that you can use as a starting point as they have the required metadata set for service discovery by the modules.
-+
-If you are deploying a Kubernetes cluster outside your local machine, setting the service to be a LoadBalancer type will give you an external IP address to connect to the redis instance.  
+The Kafka service will be used for messaging between modules in the stream.  There are sample replication controller and service YAML files in the `spring-cloud-dataflow-server-kubernetes` repository that you can use as a starting point as they have the required metadata set for service discovery by the modules.
 + 
 ```
-$ git clone https://github.com/spring-cloud/spring-cloud-dataflow-admin-kubernetes
+$ git clone https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes
 $ cd spring-cloud-dataflow-admin-kubernetes
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:21]: cd spring-cloud-dataflow-server-kubernetes

**Comment by cppwfs** [comment]
On line https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes/pull/67/files#diff-cf0f94f2ad5a40aec19a521fee0fca4aR38 
properties is misspelled

**Comment by cppwfs** [comment]
Modules section is not being rendered correctly :-(  
Is the URL for  `scsm-asciidoc` still correct,  or should we pointing to starters?

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -53,28 +51,37 @@ $ cd <redis-install-dir>
...
+$ java -jar spring-cloud-dataflow-server-kubernetes-1.0.0.M2.jar --spring.cloud.deployer.kubernetes.cpu=1
 ```
 +
 Ensure that the Data Flow Server is running in the same terminal session that has the Kubernetes environment variables set.
 +
 . Download and run the Spring Cloud Data Flow shell.
 +
 ```
-$ wget http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/1.0.0.BUILD-SNAPSHOT/spring-cloud-dataflow-shell-1.0.0.BUILD-SNAPSHOT.jar
+$ wget http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/1.0.0.M3/spring-cloud-dataflow-shell-1.0.0.M3.jar
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:75]: should snapshot be milestone?

Comment (trisberg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:75]: Ahh, yes the repo should change too

File: spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (trisberg) [inline spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc:21]: right

**Comment by cppwfs** [comment]
To fix the modules section we need to replace scsm-asciidoc and its url in the index.adoc with :scstas-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-stream-app-starters/master/spring-cloud-stream-app-starters-docs/src/main/asciidoc .
Then update the scsm-asciidoc with sctas-asciidoc in module.adoc

**Comment by cppwfs** [comment]
Will fix comments above on merge.  
Other than those LGTM

**Comment by cppwfs** [comment]
Fixed, Squashed Merged.

**Pull request closed without merging**